MyString.java

```diff
@@ -18,13 +18,40 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String lowerCase = "";
+
+        for (int i = 0; i < str.length(); i++) {
+            char ch = str.charAt(i);
+            if (ch >= 65 && ch <= 90) {
+                ch += 32;
```

Comment:
1.Use chars rather than ASCII values
2. +32 is not clear, I want you to use `ch = (char)(ch - 'A' + 'a');